schemas/target/target-classification.schema.json

```diff
@@ -0,0 +1,98 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/schemas/target-classification",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Target Classification Fields",
+  "description": "A set of meta-data fields that will classify Target activities and experiences.",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "definitions": {
+    "target": {
+      "properties": {
```

Comment:
As we discussed during our call last week, could you please add a top level object field here which will give some context to the nested properties below ? something like
`targetClassification: {
   activityName:
  experienceName
  etc.....
}
`

Reply:
The top level object is "target", if that seems too generic, I can rename it to targetClassification

Reply:
Renamed it to targetClassification